requirements-markdown-tests.txt

```diff
@@ -8,7 +8,7 @@ prefect-azure[blob_storage]
 prefect-bitbucket
 prefect-dask
 prefect-databricks
-prefect-dbt
+# prefect-dbt
```

Comment:
For some reason having this installed was causing the prefect server to fail to start. Removing for now as the dbt docs have other issues to handle.